schema/server/branches.json

```diff
@@ -0,0 +1,8 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "version": "0.0.1",
+  "type": "array",
+  "items": {
+    "$ref": "branch.json"
```

Comment:
if we're not going to reuse branch.json elsewhere you might as well inline it here?

Reply:
have changed schema around in the next PR, branch.json does get reused so ill keep this as is for now